man/nextcloudcmd.1.rst

```diff
@@ -74,18 +71,18 @@ To synchronize the nextCloud directory ``Music`` to the local directory ``media/
 through a proxy listening on port ``8080`` on the gateway machine ``192.168.178.1``,
 the command line would be::
 
-  $ nextcloudcmd —httpproxy http://192.168.178.1:8080 \
+  $ nextcloudcmd —httpproxy http://192.168.178.1:8080 --path /Music \
                 $HOME/media/music \
-                https://server/nextcloud/remote.php/dav/Music
+                https://server/nextcloud
 
 ``nextcloudcmd`` will enquire user name and password, unless they have
 been specified on the command line or ``-n`` (see `netrc(5)`) has been passed.
 
 Using the legacy scheme, it would be::
 
-  $ nextcloudcmd —httpproxy http://192.168.178.1:8080 \
+  $ nextcloudcmd —httpproxy http://192.168.178.1:8080 --path /Music \
                 $HOME/media/music \
-                nextclouds://server/nextcloud/remote.php/dav/Music
+                nextclouds://server/nextcloud
```

Comment:
I guess this `nextclouds` scheme is not working because the code is expecting `owncloud` as a scheme